src/Helpers/cookies.js

```diff
@@ -7,23 +8,26 @@ export const setCookie = (
   path?: string = '/'
 ): void => {
   const expires = new Date(Date.now() + days * 864e5).toUTCString()
-
-  document.cookie =
-    name +
-    '=' +
-    encodeURIComponent(value) +
-    '; expires=' +
-    expires +
-    '; path=' +
-    path
+  if (isBrowser) {
+    document.cookie =
+      name +
+      '=' +
+      encodeURIComponent(value) +
+      '; expires=' +
+      expires +
+      '; path=' +
+      path
+  }
 }
 
 export const getCookie = (name: string): string | null => {
-  return document.cookie.split('; ').reduce((r, v) => {
-    const parts = v.split('=')
+  if (isBrowser) {
+    return document.cookie.split('; ').reduce((r, v) => {
+      const parts = v.split('=')
 
-    return parts[0] === name ? decodeURIComponent(parts[1]) : r
-  }, null)
+      return parts[0] === name ? decodeURIComponent(parts[1]) : r
+    }, null)
+  }
```

Comment:
```suggestion
  }
  
  return null
```

Flow will appreciate it